<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_generator">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Generator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_queue">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Queue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cosumer">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Consumer</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>782</width>
      <height>194</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listGenerator"/>
     </item>
     <item>
      <widget class="QListWidget" name="listQueue"/>
     </item>
     <item>
      <widget class="QListWidget" name="listConsumer"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>121</width>
      <height>78</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_status_generator">
       <property name="font">
        <font>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>Running</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartGenerator">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStopGenerator">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>121</width>
      <height>78</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_status_consumer">
       <property name="styleSheet">
        <string notr="true">QLabel {
	qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>Running</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartConsumer">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStopConsumer">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
